### ISSUE TITLE
Business Central-not have the following permissions on TableData NAV App: IndirectRead

### ISSUE THREAD

**Issue by alopez5566** [Microsoft/AL#3592]
Good Morning;

     I did setup my VS code and Al ,, when I try to download symbols ( from production BC 365 cloud)  , I'll get error ' not have the following permissions on TableData NAV App: IndirectRead'.. I have  'super' access in the  companies , but still no success trying to extend a table.  I'm  a C# developer but new to BC with AL Code..  I did try this with a different client ( sandbox) and everything when good.. can someone guide or help me ...

Thanks;

**Comment by thpeder** [comment]
Hi @alopez5566,
Try to login to the product from the client and then go back to VSCode and download symbols/publish your app again

**Comment by kalberes** [comment]
Yes it is, you need a sandboxed environment which you can create from production.
